js/main.js

```diff
@@ -0,0 +1,53 @@
+'usestrict'
+
+{
+  const tableBody = document.getElementById('todo-body')
+  const text = document.getElementById('text');
+  const btn = document.getElementById('btn');
+  const todos = [];
+
+  document.querySelector('button').addEventListener('click', () => {
+    const todo = {};
+    todo.tableComment = text.value;
+    todo.tableStatus = '作業中'
+
+    if (todo) {
+      todos.push(todo);
+      text.value = '';
+      showTodos();
+    }
+  });
+  
+  const showTodos = () => {
+    tableBody.textContent = '';
+    todos.forEach((todo, number) => {
+      const tableRecord = document.createElement('tr');
+      tableBody.appendChild(tableRecord);
+      const tableId = document.createElement('td');
+      const Comment = document.createElement('td');
+      const Status = document.createElement('td');
+      const Action = document.createElement('td');
```

Comment:
スコープが狭い場合は `const` であっても `camelCase` で問題ないです。
```suggestion
      const comment = document.createElement('td');
      const status = document.createElement('td');
      const action = document.createElement('td');
```